### ISSUE TITLE
For grid Accessibility, cells should be read with their column heading

### ISSUE THREAD

**Issue by dlgski** [angular-ui/ui-grid#4929]
I know that @JLLeitschuh has made great progress in getting UI-Grid to be accessible.  But it is still difficult to follow with a screen reader.  As you step through cells, they should read the column header along with the cell contents.  This will make for a much better screen reader experience.

Thanks!

**Comment by mportuga** [comment]
Note: Add help wanted label

**Comment by usmanibnesadiq** [comment]
Hey guys, we are using this grid and our accessibility testing report came up with this issue that cell content is not readable with column headings. Any ideas/points if this can be fixed or has already been addressed? Thanks,

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
